tests/data_context/test_data_context_v013.py

```diff
@@ -139,25 +139,23 @@ def test_ConfigOnlyDataContext_v013__initialization(
     )
 
 
+@pytest.mark.unit
 def test__normalize_absolute_or_relative_path(
     tmp_path_factory, basic_data_context_v013_config
 ):
-    config_path = str(
-        tmp_path_factory.mktemp("test__normalize_absolute_or_relative_path__dir")
+    full_test_path = tmp_path_factory.mktemp(
+        "test__normalize_absolute_or_relative_path__dir"
     )
+    test_dir = full_test_path.parts[-1]
+    config_path = str(full_test_path)
     context = BaseDataContext(
         basic_data_context_v013_config,
         config_path,
     )
 
-    pattern_string = os.path.join(
-        "^.*test__normalize_absolute_or_relative_path__dir\\d*", "yikes$"
```

Comment:
The `\d*` suffix is a private implementation detail of how `tmp_path_factory` creates temporary directories. Instead I use the directory that `tmp_path_factory` tells us it created when we call `mktemp`.

Reply:
really appreciate this change. thank you @billdirks